docs/datasync/delta-table.md

```diff
@@ -0,0 +1,208 @@
+---
+id: delta-table
+title: Delta Tables with server-side Conflict resolution
+sidebar_label: Setting up server-side Conflict Resolution
+---
+
+This is a more involved strategy for data synchronization that includes server-side conflict resolution
+
+## Setup
+
+Feel free to grab the official Graphback template for this strategy [here](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-datasync-backend). Otherwise, It is fairly easy to set this up with the official Graphback template for MongoDB (found [here](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-backend)).
```

Comment:
```suggestion
Feel free to grab the official Graphback template for this strategy [here](https://github.com/aerogear/graphback/tree/templates-release/templates/ts-apollo-mongodb-datasync-backend). Otherwise, It is fairly easy to set this up with the official Graphback template for MongoDB (found [here](https://github.com/aerogear/graphback/templates-release/master/templates/ts-apollo-mongodb-backend)).
```

Use the templates release branch, master can be unstable.